src/main/java/org/zenoss/dataflow/metric/ingest/compact/MetricIngestCompactPipeline.java

```
@@ -127,7 +127,8 @@ public void createPipelineDAG(final Pipeline p) throws DataflowException {
 
```

The tenant change is fine, but I noticed that the pipeline will silently fail on line 120 if bigTableConfig is not set. That should not be so i.e. writing to BT is a core function.  
We should:

- Remove the null check on line 120
- Instead add a MiscUtils.validateOption() to the tableId retrieved on line 81 in configure(). That is, we need to fail the pipeline if the bigTableConfig cannot be created.